meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/resin-proxy-config

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+#
+# copy over redsocks configuration file, set proper
+# permissions and start redsocks
+#
+
+set -e
+
+. /usr/sbin/resin-vars
+
+if [ ! -f $CONFIG_PATH ]; then
+	echo "resin-proxy-config: $CONFIG_PATH does not exist."
+	exit 1
+else
+	echo "resin-proxy-config: Found config.json in $CONFIG_PATH ."
+fi
+
+REDSOCKSCONF=${CONFIG_PATH%/*}/system-proxy/redsocks.conf
+
+if [ ! -f $REDSOCKSCONF ]; then
+	echo "resin-proxy-config: No proxy configuration found, skipping."
+	exit 0
+fi
+
+cp $REDSOCKSCONF /etc/redsocks/
+chmod 600 /etc/redsocks/redsocks.conf
```

Comment:
I copied the pattern from `resin-net-config`, but I guess we could use it directly from the boot partition...